### ISSUE TITLE
3 open tickets not one answered?

### ISSUE THREAD

**Issue by Sirmadsen** [cocktailpeanut/comfyui.pinokio#4]
Developer just don't care or don't know...

How can we add custom arguments to the launch since this doesn't pull from the usual run_nvidia_gpu.bat???

**Comment by LIQUIDMIND111** [comment]
> Developer just don't care or don't know...
> 
> How can we add custom arguments to the launch since this doesn't pull from the usual run_nvidia_gpu.bat???

indeed, there are isssues since august NON replied..... maybe we should trust in this dev.

**Comment by lustfeind** [comment]
I like the idea behind Pinokio, but  I'm going back to run comfyUI the old way. On Mac silicon I don't can use fp16 models neither coreML custom nodes. Sadly, byebye.

**Comment by tetsuoo-online** [comment]
I'm here for the same reason. Pinokio is nice but all the apps installed from there would automatically launch my default browser (with all the tabs), I hate it. I prefer using an external window that I can put anywhere I want (on second monitor for example)
